# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/thebes/SharedFontList.cpp
# Commit: 277876c31869
# Full Hash: 277876c31869cad23a2e93aab7bb4c7ee0c5f0f9
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2019-08-13 09:54:32
# Regressor Bug: 1565966
# File Overlap Count: 1
# Description:
#   Bug 1565966 - part 1 - Normalize font family names to lowercase in mAliasTable, for consistency with eventual font-family lookups. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D41242
# ==============================================================================

diff -r ed6b960586ea -r 277876c31869 gfx/thebes/SharedFontList.cpp
--- a/gfx/thebes/SharedFontList.cpp	Mon Aug 12 18:23:28 2019 +0000
+++ b/gfx/thebes/SharedFontList.cpp	Mon Aug 12 17:49:09 2019 +0000
@@ -642,9 +642,7 @@
   nsTArray<Family::InitData> aliasArray;
   aliasArray.SetCapacity(aAliasTable.Count());
   for (auto i = aAliasTable.Iter(); !i.Done(); i.Next()) {
-    nsAutoCString key(i.Key());
-    ToLowerCase(key);
-    aliasArray.AppendElement(Family::InitData(key, i.Key()));
+    aliasArray.AppendElement(Family::InitData(i.Key(), i.Key()));
   }
   aliasArray.Sort();
 